source/features/extend-diff-expander.js

```diff
@@ -4,7 +4,7 @@ import delegate from 'delegate';
 function expandDiff(event) {
 	// Skip if the user clicked directly on the icon
 	if (!event.target.closest('.js-expand')) {
-		select('.js-expand', event.target.parentElement).click();
+		select('.js-expand', event.currentTarget).click();
```

Comment:
This doesn't work. `.currentTarget` is the `.diff-view` element so `select` will find the first expander of the page.

Reply:
I tested it, `delegateTarget` is diff-view and `currentTarget` is js-expandable-line

Reply:
It's not the right one. Try again. `select` picks the first element it finds. Try on a longer page and click the last expander.

Reply:
For example, click the last expander at the bottom of this page: https://github.com/sindresorhus/refined-github/pull/935/files

Reply:
Yeah you're right, I read the delegate readme incorrectly. `delegateTarget` is actually .js-expandable-line